osquery/CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ if(WINDOWS)
     set(WO_KEY "")
   endif()
   ADD_OSQUERY_LINK_CORE("ws2_32.lib")
+  ADD_OSQUERY_LINK_CORE("crypt32.lib")
```

Comment:
No you shouldn't if there are other libraries that depend on this, include this at the bottom.